frontend/components/Dashboard/DashboardScreen/index.js

```diff
@@ -137,6 +146,7 @@ const styles = StyleSheet.create({
 	contentContainer: {
```

Comment:
Seems like this isn't being used, though I would've guessed it should be... Could this be related to the styles injecting above?